### PR TITLE
XW | Fix phantomas errors

### PR DESCRIPTION
**Pull request by vforge**
Fixes for Phantomas errors:
- [x] changes inside SVGs - replace stuff like `id="a"` with unique versions
- [x] run through SVG optimizer again
- [x] inline brackets.svg

/cc @rogatty @tomasznapieralski

### CODE REVIEW

**Comment by hakubo** [comment]
LGTM

File: front/styles/main/_settings.scss [hunk 1]
```diff
@@ -1,6 +1,9 @@
 // Use Wikia style guide settings
 @import '../../vendor/wikia-style-guide/src/scss/lib/settings';
 
+// source: svg/images/brackets.svg (it's better to inline this very small file than request it)
+$brackets-svg-image: url('data:image/svg+xml;<svg xmlns="http://www.w3.org/2000/svg" width="185" height="185" viewBox="0 0 185 185"><path d="M66.8 112.894H54.84V72.167H66.8V57.353H33.146v70.292H66.8zm61.566-55.54h-10.158v14.813h11.948v40.224h-11.948v15.255h33.648v-70.29z" fill="#fff"/></svg>');
```
Comment (hakubo) [inline front/styles/main/_settings.scss]: maybe we should use something like: 
https://github.com/davidkpiano/sass-svg

Comment (vforge) [inline front/styles/main/_settings.scss]: That's great idea - I've created https://wikia-inc.atlassian.net/browse/XW-869 and I'll add this to TODO comment.

File: front/styles/main/_settings.scss [hunk 2]
```diff
@@ -1,6 +1,10 @@
 // Use Wikia style guide settings
 @import '../../vendor/wikia-style-guide/src/scss/lib/settings';
 
+// source: svg/images/brackets.svg (it's better to inline this very small file than request it)
+// @todo threat it as a temporary solution - see https://wikia-inc.atlassian.net/browse/XW-869
+$brackets-svg-image: url('data:image/svg+xml;<svg xmlns="http://www.w3.org/2000/svg" width="185" height="185" viewBox="0 0 185 185"><path d="M66.8 112.894H54.84V72.167H66.8V57.353H33.146v70.292H66.8zm61.566-55.54h-10.158v14.813h11.948v40.224h-11.948v15.255h33.648v-70.29z" fill="#fff"/></svg>');
```
Comment (rogatty) [inline front/styles/main/_settings.scss]: Shouldn't this be encoded? See `styles/main/theme/_share-icons.scss`.

File: front/svg/symbols/main/apple-store-logo.svg [hunk 3]
```diff
@@ -1 +1,13 @@
...
+<svg width="150" height="44" viewBox="0 0 150 44" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+  <defs>
+    <path id="apple-store-logo-a" d="M.067.044H150V44H.067"/>
```
Comment (rogatty) [inline front/svg/symbols/main/apple-store-logo.svg]: I don't know why but on my devbox it's compiled to `id="a"` in `www/front/svg/main.svg`
Before merging we should test it to be sure it works as intended. Maybe I missed something?

File: front/styles/main/_settings.scss (hunk 2 above)
Comment (rogatty) [inline front/styles/main/_settings.scss]: Done in https://github.com/Wikia/mercury/pull/1812

**Pull request closed without merging**